### DISCUSSION TITLE
Filter selections overlap with browser window ~990px or less

### DISCUSSION THREAD

**Discussion by ljread** [code4recovery/12-step-meeting-list#932] [Q&A] [answered]
Trying to implement the TSML UI on the webpage wp.nwpi.net/meetings. On wider browser windows the filter selection boxes behave as expected (except the list/map selection is missing.) However on smaller widths (seems to be around 990px) the last filter box overlaps (see attached screenshot.)

I have searched for similar issues in discussions but did not find anything relevant. Forgive me if I missed something.

Here are some details: Wordpress version 6.0.2; TSML version 3.14.6; php version 7.4.10. The site also uses the Elementor and Elementor Pro plugins versions 3.7.6 (both.)

Currently I have switched back to the Legacy UI as the new UI doesn't present well on mobile devices.

Any suggestions on a solution will be greatly appreciated. I think the new UI is an improvement and I would like to be able to use it.

Thanks for your service,

John

<img width="990" alt="Screenshot 2022-09-18 at 18 15 23" src="https://user-images.githubusercontent.com/113863608/190932933-d186b24a-030d-4c44-b6dc-5893872f525e.png">

**Comment by joshreisner** [comment]
i'm guessing its some CSS in your theme that is conflicting with TSML UI. if we could see it we could recommend some fixes. could you create a new secret page and use the [tsml_ui] shortcode?

**Comment by ljread** [comment]
Thanks Josh for the quick response. The site theme is 2017.

Pretty sure I don't know how to create a second meeting archive page. So instead I just switched the live page to the new UI. It's not pretty but can still be used.

The lack of the list/map button was due to no Mapbox token. I added a token and now it appears.

Thanks again for your help

**Comment by joshreisner** [comment, marked as answer]
Ah ok @ljread here is some CSS to add to your theme:

```
#tsml-ui .dropdown-toggle {
  position: static;
}
```

**Comment by ljread** [comment]
@joshreisner, that fixed it. Most excellent. Appreciate your help and all that Code for Recovery did with TSML plugin during covid. We'd never have kept up with the meeting changes without you guy.

Thanks for your service.
